docs/auth/README.md

```diff
@@ -25,7 +25,33 @@ For example, let's show how to provide **Read Only** access to our `orders` Data
 
 Using the automatically generated API, we can now interact with our data. 
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+from jexia_sdk.http import HTTPClient
+JEXIA_PROJECT_ID = ''
+JEXIA_API_KEY = ''
+JEXIA_API_SECRET = ''
+if __name__ == '__main__':
+  client = HTTPClient()
+  client.auth_consumption(
+      project=JEXIA_PROJECT_ID,
+      method='apk',
+      key=JEXIA_API_KEY,
+      secret=JEXIA_API_SECRET,
+  )
+  res = client.request(
+          method='GET', 
+          url='/ds/orders',
+          cond='[{"field":"dislike"},"=",true]',
+          outputs='["id","total","title"]'
+        ) 
+  print(res)
+  
```

Comment:
will be better to use pep8 formating for code examples:
1. use 4 spaces everywhere
2. use empty line between modules and if section
3. use correct number of spaces

So this example should looks like:

```python
from jexia_sdk.http import HTTPClient

JEXIA_PROJECT_ID = ''
JEXIA_API_KEY = ''
JEXIA_API_SECRET = ''

if __name__ == '__main__':
    client = HTTPClient()
    client.auth_consumption(
        project=JEXIA_PROJECT_ID,
        method='apk',
        key=JEXIA_API_KEY,
        secret=JEXIA_API_SECRET,
    )
    res = client.request(
        method='GET',
        url='/ds/orders',
        cond='[{"field":"dislike"},"=",true]',
        outputs='["id","total","title"]'
    )
    print(res)
```

Reply:
fixed.